content/docs/manage/orgs-project-transfer.md

```diff
@@ -15,6 +15,7 @@ A few important points to keep in mind:
 - You must be at least a Member of the selected Organization to transfer projects to it.
 - The number of projects you can transfer is limited by the target Organization plan's allowance.
 - Projects can't be transferred between incompatible plans due to differences in usage allowances. For example, attempting to transfer projects from a Scale plan personal account to a Launch plan Organization will result in an error.
```

Comment:
Should we merge this line into the new one below? Both mention the same restriction, but the new one sounds more generic.

Reply:
Thank you. Merged.